webpack.config.js

```diff
@@ -156,6 +156,35 @@ module.exports = {
                 exclude: /node_modules/,
                 loader: extract.extract('css-loader!sass-loader'),
                 test: /\.scss$/
+            },
+            {
+                test: /\.js$/,
+                exclude: /\/node_modules\/.*\//, // exclude external dependencies
+                loader: 'strict-loader' // Adds "use strict"
+            },
+            {
+                test: /\.jsx$/,
+                loader: "babel-loader"
+            },
+            {
+                test: /\.es6$/,
+                loader: "babel-loader"
+            },
+            {
+                test: /\.css$/,
+                loader: extract.extract("css-loader?minimize=&root=" + libdir)
+            },
+            {
+                test: /\.less$/,
+                loader: extract.extract("css-loader?sourceMap&minimize=!less-loader?sourceMap&compress=false")
+            },
+            {
+                test: /views\/[^\/]+\.html$/,
+                loader: "ng-cache?prefix=[dir]"
+            },
```

Comment:
This looks unrelated and should be a separate commit with a proper justification.